sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -10,6 +10,9 @@
 - Added stricter behavior for ArmStr schemas when parsing 'azureml:' prefix.
 - Improved intellisense with VS Code for fields supporting local paths and datastores.
 
+### Other Changes
+- Removed dependency on API version 2021-10-01 and 2022-06-01-preview to reduce side of azure-ai-ml package.
```

Comment:
do we know how much of a size reduction there was?

Reply:
Let's see. :)